homework/src/style.css

```diff
@@ -1,3 +1,134 @@
+body {
```

Comment:
While you could argue that your app is responsive as it adjusts to the screen width, I think most people would expect the contributor list to be placed below the repo info rather than next to it in the case of a portrait mobile display.

This is would your app would look like on an iPhone 6:

![responsive](https://user-images.githubusercontent.com/2788771/44986871-7009c600-af85-11e8-8dc3-d3f28052ba38.png)